backend/packages/Upgrade/src/loaders/app/index.ts

```diff
@@ -5,7 +5,6 @@ import { authorizationChecker } from '../../auth/authorizationChecker';
 import { currentUserChecker } from '../../auth/currentUserChecker';
 
 const expressApp: Application = createExpressServer({
-  cors: true,
   classTransformer: true,
```

Comment:
`cors` options should be directly supplied here instead of making it a middleware because it will ensure that it is the first middleware getting used in the server which it should be.